### PR TITLE
list view: further implement the list view

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
closes https://github.com/inveniosoftware/invenio-administration/issues/6

### CODE REVIEW

File: invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js [hunk 1]
```diff
@@ -6,44 +6,82 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import React from "react";
-import { Input } from "semantic-ui-react";
-import { i18next } from "@translations/invenio_communities/i18next";
+import React, { Component } from "react";
+import { i18next } from "@translations/invenio_search_ui/i18next";
```
Comment (jennur) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js:10]: Just make sure this gets updated when https://github.com/inveniosoftware/invenio-administration/issues/17 is merged :)

**Review by jennur — APPROVED** [review]
LGTM!

File: invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js [hunk 2]
```diff
@@ -6,44 +6,78 @@
...
-        updateQueryState({ queryString });
-      }
+export class SearchBarComponent extends Component {
+  constructor(props) {
+    super(props);
+
+    const { queryString } = props;
+
+    this.state = {
+      currentQuery: queryString || "",
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js:22]: Could be set in component's `defaultProps`

File: invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js [hunk 3]
```diff
@@ -6,44 +6,78 @@
...
+  queryString: PropTypes.string,
+  updateQueryState: PropTypes.func.isRequired,
+  currentQueryState: PropTypes.object.isRequired,
+  placeholder: PropTypes.string.isRequired,
+  uiProps: PropTypes.object,
+};
+
+SearchBarComponent.defaultProps = {
+  uiProps: undefined,
+  placeholder: i18next.t("Search ..."),
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/search/SearchBarElement.js:80]: (from CI actions): 

`placeholder` is required, does not need a `default`.

File: invenio_administration/assets/semantic-ui/js/invenio_administration/search/search.js [hunk 4]
```diff
@@ -9,38 +9,45 @@
...
 import { SearchResults } from "./SearchResults";
 import { parametrize } from "react-overridable";
 import { DropdownSort } from "@js/invenio_search_ui/components";
+import { SearchResultOptions } from "./SearchResultOptions";
+import { SearchBar } from "./SearchBarElement";
 
 const domContainer = document.getElementById("invenio-search-config");
+
+const sortColumns = (columns) =>
+  Object.entries(columns).sort((a, b) => a[1].order > b[1].order);
```
Comment (alejandromumo) [inline invenio_administration/assets/semantic-ui/js/invenio_administration/search/search.js:23]: If `order` is not defined in the entries, would this throw an exception in the browser? E.g. defining `order` to something silly.

**Review by alejandromumo — COMMENTED** [review]
Blocked by https://github.com/inveniosoftware/invenio-administration/pull/20/files#r957440867 

Note also that there are some tests failing in the CI due to eslint (e.g. missing commas or semicolons)

**Pull request merged**